packages/grid_client/src/modules/tft.ts

```diff
@@ -0,0 +1,53 @@
+import { QueryClient } from "@threefold/tfchain_client";
+import Decimal from "decimal.js";
+
+import { expose, validateInput } from "../helpers";
+import { calculator as Calculator } from "./calculator";
+import { TFTModel, USDModel } from "./models";
+
+enum substrateURL {
+  DEV = "wss://tfchain.dev.grid.tf/ws",
+  QA = "wss://tfchain.qa.grid.tf/ws",
+  TEST = "wss://tfchain.test.grid.tf/ws",
+  MAIN = "wss://tfchain.grid.tf/ws",
+}
+
+class TFT {
+  public SUBSTRATE_URL: string;
+  private calculator: Calculator;
+
+  constructor(SUBSTRATE_URL: string) {
+    this.SUBSTRATE_URL = SUBSTRATE_URL;
+    this.calculator = new Calculator(new QueryClient(this.SUBSTRATE_URL));
+  }
+
+  async price() {
+    return await this.calculator.tftPrice();
+  }
+
+  @expose
+  @validateInput
+  async fromUSD(options: USDModel) {
+    return new Decimal(options.usd / (await this.price())).toFixed(2);
+  }
+
+  @expose
+  @validateInput
+  async toUSD(options: TFTModel) {
+    return new Decimal(options.tft * (await this.price())).toFixed(2);
+  }
+  @expose
+  @validateInput
+  toMonth(options: TFTModel) {
```

Comment:
change to `monthlyTFT` and change the parameter name to `hourlyTFT` and don't miss the return type

Reply:
and in the comments point out that it will be calculated in 30 days and let's add dailyTFT